cart/domain/cart/cart.go

```diff
@@ -601,6 +628,30 @@ func (t Taxes) TotalAmount() domain.Price {
 	return result
 }
 
+// GetAllTaxes – returns all Taxes as slice
+func (t Taxes) GetAllTaxes() []Tax {
```

Comment:
Taxes is already a slice of Tax:

Taxes []Tax

you can range over the variable taxes itself - no need for this Getter

Reply:
This one is tricky in graphql, truly Taxes could be a slice of Taxes but then it's not possible to access getter methods of Taxes type like the TotalAmount or GetByType. Maybe i'm missing something?

Reply:
I'm also unsure about the get all methods, in other places we have sub property for items, like the PricedItems.